tests/test_TextFilter.py

```diff
@@ -0,0 +1,403 @@
+#!/usr/bin/env python
```

Comment:
Follow the PEP 8 style here too. Pycharm allows you to step through all warnings with f2